### ISSUE TITLE
No way to mute notification sounds or change volume in 1.0 on OSX

### ISSUE THREAD

**Issue by ssvenn** [cgrossde/Pullover#59]
It would be very nice to have more control over notification sounds, like adjusting the volume/muting or overriding the default sound file locally

Normally it wouldn't be a big deal but the default notification sounds enough like a doorbell that my dog thinks we're getting visitors each time it dings, and her visible disappointment after running to the door is really heartbreaking!

**Comment by cgrossde** [comment]
:D I think this is a good idea. Not sure about the volume yet but disabling/changing the default sound will definitely come in a future version.

**Comment by Vitassam** [comment]
Yeah, I also can't disable sound of my notifications in this new version.
Thank you for your development.
